locales/en/messages.json

```diff
@@ -3490,17 +3490,17 @@
         "message": "LEDs without wire ordering number will not be saved.",
         "description": "Message in the wiring modes in Led Strip"
     },
-    "ledStripVtxFunction": {
+    "ledStripLarsonOverlay": {
         "message": "Larson scanner"
     },
-    "ledStripBlinkTitle": {
-        "message": "Blink"
-    },
```

Comment:
Why is this deleted? I suppose now is the new `ledStripModifierTitle`?

Reply:
It's not used anymore.
Before:
![Screenshot_2023-02-10-13-28-35-140-edit_com betaflight betaflight_configurator_debug](https://user-images.githubusercontent.com/62965528/218092815-88e8d717-bef2-4673-8f68-bb4d01c42c32.jpg)

After:
![Screenshot_2023-02-10-13-27-45-971-edit_com betaflight betaflight_configurator_debug](https://user-images.githubusercontent.com/62965528/218092842-fac0ce45-e99b-4cef-8656-327f3f1dd7c6.jpg)